frontend/packages/ceph-storage-plugin/src/utils/install.ts

```diff
@@ -113,12 +113,31 @@ export const countNodesPerZone = (nodes: NodeKind[]) =>
 export const nodesWithoutTaints = (nodes: NodeKind[]) =>
   nodes.filter((node: NodeKind) => hasOCSTaint(node) || hasNoTaints(node));
 
+export const isNormalSC = (
+  sc: StorageClassResourceKind,
+  pvData: K8sResourceKind[],
+  nodesData: NodeKind[],
+  scName?: string,
+): boolean => {
+  const pvs: K8sResourceKind[] = getSCAvailablePVs(pvData, scName || getName(sc));
+  const scNodeNames = getAssociatedNodes(pvs);
+  const tableData: NodeKind[] = nodesData.filter(
+    (node: NodeKind) =>
+      scNodeNames.includes(getName(node)) ||
+      scNodeNames.includes(node.metadata.labels?.['kubernetes.io/hostname']),
+  );
+  const uniqSelectedNodesZones: Set<string> = new Set(tableData.map(getZone));
+  if (_.compact([...uniqSelectedNodesZones]).length < 3) return false;
```

Comment:
There is a helper function `getNodeInfo` I think we can use that instead of later using `_.compact` to remove values.

Reply:
function `getNodeInfo` only considers nodes which `hasNoTaints`... so, if I have created a storage-cluster with say 3 nodes () and now I am adding-capacity with the same storage-class... it will show error and won't allow to expand...

Reply:
okay